R/calculate.R

```diff
@@ -38,7 +38,8 @@ calculate_annual_profits <- function(asset_type, input_data_list, scenario_to_fo
       scenario_data = input_data_list$scenario_data,
       start_analysis = start_year,
       end_analysis = end_year,
-      time_frame = time_horizon
+      time_frame = time_horizon,
+      target_scenario = shock_scenario_lookup
```

Comment:
could you explain again what the difference between target and shock scenario is? In the results for company_trajectories they are 2 different variables, practically they are the same I guess and here the naming is mixed>